pom.xml

```diff
@@ -334,6 +335,10 @@
               <rules>
                 <enforceBytecodeVersion>
                   <maxJdkVersion>${compileSource}</maxJdkVersion>
+                  <ignoreClasses>
+                    <ignoreClass>module-info</ignoreClass>
```

Comment:
Note the jira key for the bug that means we have to exclude module info

Reply:
Changed to 'Ignore module-info and SubmissionPublisher in enforceBytecodeVersion rule'